python/publish/publisher.py

```diff
@@ -263,6 +264,7 @@ def publish_check(self,
             annotations=all_annotations
         )
         self.publish_json(data)
+        self.publish_summary(data)
```

Comment:
I'd prefer to use `title` and `summary` explicitly when calling into `publish_summary`. Move that call right after `summary = get_long_summary_md(stats_with_delta)`.

Reply:
As you wish, I was making it that way in case we'd wish to extend thé summary with more data

Reply:
I prefer to keep comment, check and job summary identical. Will adjust if really needed.